packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -75,6 +75,8 @@ export class KvTextField implements ITextField, ITextFieldEvents {
 	@Prop({ reflect: true }) useInputMask?: boolean = false;
 	/** @inheritdoc */
 	@Prop({ reflect: true }) inputMaskRegex?: string = '';
+	/** @inheritdoc */
+	@Prop({ reflect: true }) resizable?: boolean = true;
```

Comment:
Maybe `fitContent` is a better name. Resizable sounds like the user can resize the input with the cursor 😄